app/assets/stylesheets/main.scss

```diff
@@ -10,6 +10,10 @@
 @import 'components/notifications';
 @import 'views/home';
 @import 'views/sign_up';
+<<<<<<< HEAD
+@import 'views/home_show';
+=======
```

Comment:
Check your code, conflict tags appear, which you should remove:

```
<<<<<<< HEAD
=======
```

Reply:
Changes made